concent_api/core/tests/utils.py

```diff
@@ -616,3 +616,10 @@ def _assert_stored_message_counter_not_increased(self):
 
     def _assert_client_count_is_equal(self, count):
         self.assertEqual(Client.objects.count(), count)
+
+    # TODO: Merge with '_sign_message' after authentication is merged
+    def _get_signature(self, golem_message, priv_key, pub_key):  # pylint: disable=no-self-use
```

Comment:
Calling it `get_xxx` is misleading because the function modifies the message it gets as input.